sol/private/versions.bzl

```diff
@@ -1,963 +1,984 @@
 "Mirrored release data from binaries.soliditylang.org, generated from tools/mirror_releases.sh"
-LATEST_VERSION = "0.8.17"
-TOOL_VERSIONS = {}
-
+LATEST_VERSION="0.8.19"
```

Comment:
this is an older pattern, we now just trust the older of keys e.g. https://github.com/aspect-build/rules_swc/blob/d784466a66e7d95e463d620e9b714ec47cbfeb3e/swc/repositories.bzl#L11

Reply:
If we were to do this we'd be relying on an undocumented invariant (list ordering) in `https://binaries.soliditylang.org/$PLATFORM/list.json` files instead of their explicit field called `latestRelease`. I'm always a strong proponent of being explicit.

Reply:
Yeah all our other repos have a comment "put this in order" (maybe we could use buildifier `#keep sorted` too) and the mirror script repeats "put this at the end". It does rely on code review to catch it done wrong, but so does this.